examples/python/FullScriptSupport_Pandas/README.md

```diff
@@ -0,0 +1,45 @@
+# Example: Full script support using Pandas
+This example plugin includes support for all script functionality and is based on the original [Full Script Support](../FullScriptSupport/README.md) Python example. The implementation of this plugin differentiates mainly by the use of the Pandas library, the data received from Qlik is now saved to a Pandas data frame. Another difference is that we now use the `exec` method to evaluate the script rather than the `eval` method, as we did in the original example plugin. This change opens up the possibility to for instance pass a multiline script from Qlik.
+
+## Content
+* [Implementation](#implementation)
+    * [Parameters sent from Qlik](#parameters-sent-from-qlik)
+    * [TableDescription](#tabledescription)
+    * [Result](#result)
+* [Qlik Documents](#qlik-documents)
+* [Run the Example!](#run-the-example)
+
+## Implementation
+We aim for a well documented code for you to easily be able to follow along. If something is unclear, please let us know so that we can update and improve our documentation. In this file we will guide you through a few specific key points in the implementation, that is worth clarifying.
+
+### Parameters sent from Qlik
+The parameters sent from Qlik are now stored in a `pandas.DataFrame` object called `q`. The names of the parameters, and hence the column names of `q`, are set to the names sent from Qlik in the _ScriptRequestHeader_. For instance if you send a parameter called `Foo` in Qlik, you will reach the parameter by writing `q.Foo` or `q["Foo"]` in the script.
+
+If the parameter is of type _Dual_ the plugin will create two additional columns in the `q` data frame, with the string and numerical representation. The column names will have the base as the parameter name but will end with '_str' and '_num' respectively. For example, a parameter called `Bar` with datatype _Dual_ will result in three columns in `q`: `Bar`, `Bar_str` and `Bar_num`. `Bar` will contain strings and numerics, `Bar_str` will contain only strings and `Bar_num` only numerics.
+
+### TableDescription
+In the load script, when using the `Load ... Extension ...` syntax you can create the `TableDescription` message within the script. This can be useful if you for instance want to name, set tags, or change datatype of the fields you are sending back to Qlik. Read more about what metadata can be included in the `TableDescription` in in the [SSE_Protocol.md](../../../docs/SSE_Protocol.md).
```

Comment:
This can be useful if, for example, you want to name, set tags for, or change the datatype of the fields you are sending back to Qlik.